lib/index.js

```diff
@@ -1,14 +1,31 @@
 'use strict';
 
+const Joi = require('joi');
+
+const internals = {
+    defaults: {
+        ver: process.env.npm_package_version,
```

Comment:
We should add defaults for both of these, and also tests for when the values are not set